### DISCUSSION TITLE
Make contributions more easy

### DISCUSSION THREAD

**Discussion by JackyWoo** [pocoproject/poco#3924] [General]
@obiltschnig @aleks-f When I was new, I found it not easy to contibute to `Poco`. Below is what bothers me. 

1. Commuity is a little slow.
2. When add some file I must adjust vs project file.

I have no ideas for the issues. Expect your answer and hope `Poco` more popular.

**Comment by aleks-f** [comment]
1. It is what it is - we do our best to maintain it in addition to our other activities
2. There is nothing we can do to avoid the necessity of adding source files to VS project files; to help with that, there is [ProGen](https://github.com/pocoproject/poco/tree/devel/ProGen) and there are cmake files

**Reply by JackyWoo** [reply]
Thanks for reply and appreciate for your work.
